src/main/scala/org/ergoplatform/nodeView/wallet/persistence/WalletRegistry.scala

```diff
@@ -252,7 +252,7 @@ class WalletRegistry(store: LDBVersionedStore)(ws: WalletSettings) extends Score
 
     // and update wallet digest
     updateDigest(bag3) { case WalletDigest(height, wBalance, wTokensSeq) =>
-      if (height + 1 != blockHeight) {
+      if (height + 1 != blockHeight && !walletCreated) {
         log.error(s"Blocks were skipped during wallet scanning, from $height until $blockHeight")
```

Comment:
This is the only reason why the walletPhase is being propagated down the `WalletRegistry` and `WalletScanLogic`